packages/tinacms/src/internalClient/index.ts

```diff
@@ -574,15 +587,13 @@ export class LocalClient extends Client {
   }
 
   async authenticate() {
+    console.log('not working authenticate')
```

Comment:
@logan-anderson do we want to keep this?

Reply:
@kldavis4 I can remove the console logs